default/scripting/buildings/colonies/col_bld_gen.py

```diff
@@ -228,7 +227,7 @@
 for species in chain((x[0] for x in species_list), species_extinct_techs.keys()):
     name = species[3:]
     exclude_parallel_colonies += f"""\
-        Not Contains Building name = "BLD_COL_{name}"
+        Number high = 0 condition = And [ Building name = "BLD_COL_{name}" ContainedBy Source VisibleToEmpire empire = Source.Owner ]
```

Comment:
the VisibleToEmpire should not be necessary. It should check for ownership here instead.

Reply:
Actually that idea occurred to me afterwards as well. At least at the moment, it doesn't make a difference and certainly never for colony buildings.